docs/reference/config.rst

```diff
@@ -167,20 +167,38 @@ Defaults to ``yes``.
 list_format_item
 ~~~~~~~~~~~~~~~~
 
+Deprecated option, replaced by :ref:`format_item`.
+
+.. _list_format_album:
+
+list_format_album
+~~~~~~~~~~~~~~~~~
+
+Deprecated option, replaced by :ref:`format_album`.
```

Comment:
We can probably remove these legacy options from the docs to avoid clutter. (It could perhaps be easiest to leave the labels in place, of course, to avoid breaking links.)